release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2997,6 +2997,8 @@ link:https://access.redhat.com/solutions/6964873[{product-title} 4.10.20 contain
 
 * Previously, Bond Container Network Interface (CNI) version 1.0 was not compatible with the Multus Container Network Interface (CNI) plug-in. Consequently, the Bond-CNI IP address management (IPAM) improperly populated the `network-status` annotation. With this update, IPAM and Bond-CNI now supports Bond-CNI 1.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084289[*BZ#2084289*])
 
+* Before this update, the Pipeline start form was always created using a hardcoded value, gp2, as the default storageclass. With this update, the Pipeline start form uses the correct storageclass name instead of the hardcoded value, gp2.
```

Comment:
```suggestion
* Before this update, the Pipeline start form was always created using a hard-coded value, `gp2`, as the default storage class. With this update, the Pipeline start form uses the correct storage class name instead of the hard-coded value, `gp2`.
```
- I'm not finding "Pipeline start form" or similar expressions anywhere in openshift-docs repo. I'm concerned whether this is the correct and standard description of this feature. 
- SSG: "A "hard-coded" value is a value that is configured in the source code such that it cannot be altered without modifying the code."
- Red Hat prefers using "storage class" over "`storageclass`" in this content because you are not referring to a specific API object, as covered in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references.

Reply:
@Dhruv-Soni11 You can use the term "Start Pipeline page" instead of "Pipeline start form"

Reply:
Thank you @sanketpathak for providing the feedback so quickly. I have gone ahead and incorporated the changes. @rolfedh kindly view the changes and let me know if it is fine.